node/declaration.ts

```diff
@@ -28,40 +28,25 @@ export const mergeDeclarations = (
   userFunctionsConfig: Record<string, FunctionConfig>,
   internalFunctionsConfig: Record<string, FunctionConfig>,
   deployConfigDeclarations: Declaration[],
-  featureFlags: FeatureFlags = {},
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
What's the goal here?

Reply:
`mergeDeclarations` is public API (used in the CLI), so removing the param is technically a breaking change. And who knows maybe we are going to need it in the future again?

but can also remove it.

Reply:
Got it, makes sense!